docassemble/ALToolbox/al_income.py

```diff
@@ -1317,3 +1333,37 @@ def net_total(
         ) - self.deduction_total(
             times_per_year=times_per_year, source=source, exclude_source=exclude_source
         )
+
+      
+class ALItemizedJobMonth(ALItemizedJob):
+    """
+    """
+    def init(self, *pargs, **kwargs):
+        kwargs['add_months'] = kwargs.get('add_months', False)
+        kwargs['is_hourly'] = kwargs.get('is_hourly', False)
+        kwargs['times_per_year'] = kwargs.get('times_per_year', 1)
+        super().init(*pargs, **kwargs)
+        
+        self.to_add.there_are_any = True
+        self.to_subtract.there_are_any = True
+
+      
+class ALItemizedJobMonthList(ALItemizedJobList):
+    """
+    """
+    def init(self, *pargs, **kwargs):
+        kwargs['source'] = kwargs.get('source', "months")
+        kwargs['object_type'] = kwargs.get('object_type', ALItemizedJobMonth)
+        kwargs['ask_number'] = kwargs.get('ask_number', True)
+        kwargs['target_number'] = kwargs.get('target_number', 12)
+        
+        kwargs['add_months'] = kwargs.get('add_months', False)
+        super().init(*pargs, **kwargs)
+        
+        month_names = [
+            "january", "february", "march", "april", "may", "june",
```

Comment:
Assuming we really do need a month, should this just be an integer to simplify translation? Can use `format_date` to display it as text in the interface. I'm not usually seeing people store months as a list of hardcoded strings.